webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -441,7 +441,7 @@ class ClassicTechniqueWriter(basePath : String, parameterTypeService: ParameterT
   def needReportingBundle(technique : EditorTechnique, methods : Map[BundleName, GenericMethod]) = technique.methodCalls.exists(elemNeedReportingBundle(methods, Nil))
 
   def canonifyCondition(methodCall: MethodCall, parentBlock : List[MethodBlock]) = {
-    formatCondition(methodCall,parentBlock).replaceAll("""(\$\{[^\}]*})""","""",canonify("$1"),"""")
+    formatCondition(methodCall,parentBlock).replaceAll("""(\$\{[^}]*})""","""",canonify("$1"),"""")
```

Comment:
why do you change this method ?

Reply:
you are breaking the replacement of variable

Reply:
Are you sure? I think `}` doesn't need to be escaped in `[]` in a regex, and no test break because of that.